freqtrade/freqtradebot.py

```diff
@@ -73,7 +73,7 @@
 class FreqtradeBot(LoggingMixin):
     """
     Freqtrade is the main class of the bot.
-    This is from here the bot start its logic.
+    This is where the bot starts its logic.
```

Comment:
while the above change is somewhat valid - i did like previous wording for this line better.

Doesn't necessarily need a full reversal - but the new sentence is oddly worded.